prometheus-puppetdb/templates/deployment.yaml

```diff
@@ -24,7 +24,29 @@ spec:
         - name: {{ .Chart.Name }}
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
-          args: ["--output", "configmap", "--namespace", "puppet"]
+        {{- if .Values.prometheusPuppetdb.args }}
+          args:
```

Comment:
Maybe we can force --output configmap so that we don't hjave to redefine it every time we override the default

Reply:
Yes and we can remove the namespace args